### ISSUE TITLE
Latest wikipedia_en_all_mini is 57G instead of 13G

### ISSUE THREAD

**Issue by Popolechien** [openzim/zim-requests#697]
By the look of it I'd say that https://download.kiwix.org/zim/wikipedia/wikipedia_en_all_mini_2023-07.zim now is a regular `nopic` version: https://library.kiwix.org/viewer#wikipedia_en_all_mini_2023-07

@kelson42 could this be a regression?
@RavanJAltaie Did you change anything to that recipe over the past month?

**Comment by kelson42** [comment]
Yes, probably you have the whole artivle and not only the first section

**Comment by RavanJAltaie** [comment]
@Popolechien no, didn't touch this recipe at all.

**Comment by kelson42** [comment]
@Popolechien @RavanJAltaie This a regression linked to the deprecation of the Wikimedia `mobile-section` API end-point. A few days ago, we secured to put and exception for our scraper to have a grace period, see https://phabricator.wikimedia.org/T340036.

That said we need to fix MWoffliner, therefore I will move the ticket to openzim/mwoffliner.
